### DISCUSSION TITLE
Is it possible to modularize a vimrc file into multiple files for the sake of readability?

### DISCUSSION THREAD

**Discussion by Sebastian-Nielsen** [JetBrains/ideavim#614] [Q&A]
As the title says: Is it possible to modularize a vimrc file into multiple files for the sake of readability?

In other words, the main .vimrc file will import other .vimrc files.

**Comment by ljchen1** [comment]
@Sebastian-Nielsen The IdeaVIM use `~/.ideavimrc` for configuration. You could use `source` command (e.g.`source ~/.any_vimrc_file`) in `~/.ideavimrc` to load multi config files.

See here for details:
https://vimdoc.sourceforge.net/htmldoc/repeat.html#:source
